### PR TITLE
Core 717 lbc mfi hero

### PR DESCRIPTION
**Pull request by roger-in-kiva**

### CODE REVIEW

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+		<div class="tw-flex tw-items-center tw-gap-1 tw-px-2 lg:tw-px-0">
+			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="">
+			<h3 class="tw-text-h3 tw-text-primary tw-whitespace-nowrap">
+				Field Partner Spotlight
+			</h3>
+			<hr class="tw-border-tertiary tw-w-full tw-ml-2 tw-hidden lg:tw-block">
+		</div>
+		<div class="tw-mt-2 lg:tw-mt-4 tw-px-2">
+			<h2 class="tw-text-h2 tw-text-primary">
+				Fundación Pro Mujer
```
Comment (dyersituations) [inline src/components/LoansByCategory/MFIRecommendations/MFIHero.vue:12]: It looks like this addition to the page just focused on a single static partner and is not behind an experiment. Is that expected? I would have assumed we'd want the data to be more dynamic if it wasn't an experiment.

**Comment by dyersituations** [comment]
Bug: I'm seeing some format issues in narrow browser windows.
image

**Comment by dyersituations** [comment]
Observation: this is the only text on the page with a tabbed first line. Is that expected?
image

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue [hunk 2]
```diff
@@ -0,0 +1,98 @@
+<template>
+	<div class="tw-w-full">
+		<div class="tw-flex tw-items-center tw-gap-1 tw-px-2 lg:tw-px-0">
+			<img class="tw-w-4 lg:tw-w-6" src="@/assets/images/mfi_recommendations/earth.svg" alt="">
```
Comment (dyersituations) [inline src/components/LoansByCategory/MFIRecommendations/MFIHero.vue:4]: I notice some blank `alt` tags in this template. We might want to fill these.

**Comment by mcstover** [comment]
@roger-in-kiva Def look forward to the polish Casey has pointed out. I want to wait to merge this until there is an experiment gating the visibility of this new section.

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue (hunk 1 above)
Comment (roger-in-kiva) [reply src/components/LoansByCategory/MFIRecommendations/MFIHero.vue:13]: Thanks @dyersituations !
Experiment condition was left, fixed now!

**Comment by roger-in-kiva** [comment]
> Bug: I'm seeing some format issues in narrow browser windows. image

Fixed, thanks!

**Comment by roger-in-kiva** [comment]
> Observation: this is the only text on the page with a tabbed first line. Is that expected? image

Yeah! It comes from design.

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue (hunk 2 above)
Comment (roger-in-kiva) [reply src/components/LoansByCategory/MFIRecommendations/MFIHero.vue]: They were filled, thanks!

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+				Fundación Pro Mujer
+			</h2>
+			<h6 class="tw-text-subhead tw-text-primary">
+				15 years with Kiva, 52,908 borrowers supported
+			</h6>
+		</div>
+		<div class="tw-flex tw-flex-col lg:tw-flex-row tw-my-3">
+			<div class="md:tw-w-2/3 md:tw-px-2 tw-relative">
+				<!-- eslint-disable max-len -->
+				<img class="md:tw-rounded" src="@/assets/images/mfi_recommendations/fundacionpromujer.jpg" alt="fundacionpromujer">
```
Comment (mcstover) [inline src/components/LoansByCategory/MFIRecommendations/MFIHero.vue:22]: Should this image have our tw-rounded-sm class applied?

**Review by mcstover — APPROVED** [review]

**Comment by dyersituations** [comment]
Bug: I'm still seeing alignment issues on smaller browser windows:
image

File: src/components/LoansByCategory/MFIRecommendations/MFIHero.vue (hunk 3 above)
Comment (roger-in-kiva) [reply src/components/LoansByCategory/MFIRecommendations/MFIHero.vue:22]: There is not design for that size, but just mobile is considering no-rounded image.

**Comment by roger-in-kiva** [comment]
> Bug: I'm still seeing alignment issues on smaller browser windows: image

That alignment is correct in mobile design, I just added a full width to the image to consider the case you are showing based on design.

**Review by dyersituations — APPROVED** [review]

**Pull request merged**